### PR TITLE
Add a comment about the default value of --threads

### PR DESCRIPTION
**Pull request by Detegr**
I was wondering what the default value was and thought it would make sense to add it in the help description.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -419,7 +419,8 @@ fn main() {
                         .long("threads")
                         .short("j")
                         .takes_value(true)
-                        .help("Set number of threads to use for searching"))
+                        .help("Set number of threads to use for searching\n\
+                               (default: The number of CPU cores on the system)"))
```
Comment (sharkdp) [inline src/main.rs]: Minor nitpick: I'd like to keep the width of the help text as small as possible. What do you think about `(default: number of available CPU cores)`?

**Comment by sharkdp** [comment]
Good suggestion, thank you!

File: src/main.rs (hunk 1 above)
Comment (Detegr) [reply src/main.rs]: That's perfect, I'll update the PR.

**Comment by sharkdp** [comment]
Thanks!

**Pull request merged**